### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by csaftoiu**
I didn't notice the demo page at first - and it's much easier to look at the demo to figure out how to use ng-table at a glance - so I thought to make it more prominent. Plus other minor fixes which may be more aesthetic than objective.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +42,7 @@ html table and eventually make it paginated so we don't have to scroll like a ma
   </tbody>
 </table>
 ```
-[result](http://samu.github.io/angular-table/walkthrough/1.html)
+[Result](http://samu.github.io/angular-table/walkthrough/1.html).
```
Comment (samu) [inline README.md]: I don't agree using a dot. Please remove them and i'll merge, thanks!

**Comment by samu** [comment]
Thanks for your inputs. Please resolve the comment i added so i can merge.

**Comment by csaftoiu** [comment]
Done!

**Comment by samu** [comment]
Awesome, thanks

**Pull request merged**